src/components/Grid/Col.tsx

```diff
@@ -112,6 +112,10 @@ interface ColProps {
     xxl?: AutoColumnWidth;
 }
 
+/**
+ * This is a test
+ * @param props
+ */
 const Column : React.SFC<ColProps & React.HTMLAttributes<HTMLDivElement>> = ({ children, className, last, lg, md, pullSm, pullMd, pullLg, pullXl, pullXxl, pushSm, pushMd, pushLg, pushXl, pushXxl, sm, xl, xxl, ...attrs }) => {
```

Comment:
Do we want to rename the function for consistency?

Reply:
Sure, I just made that change.